tests/IndexTest.php

```diff
@@ -582,16 +582,17 @@ public function testCreate(): void
         $indexName = 'test';
 
         $index = $client->getIndex($indexName);
-        $index->create([]);
-        $this->_waitForAllocation($index);
+        $index->create([], [
+            'wait_for_active_shards' => 'all',
```

Comment:
Here's a concrete example of the use of native options when creating an index.
This allows to replace the use of `$this->_waitForAllocation()` by a native option.